Formula/micronaut.rb

```diff
@@ -1,15 +1,22 @@
 class Micronaut < Formula
   desc "Modern JVM-based framework for building modular microservices"
   homepage "https://micronaut.io/"
-  url "https://github.com/micronaut-projects/micronaut-starter/archive/v2.0.0.RC1.tar.gz"
-  sha256 "91dc71b200c598a91b95a20792dcc3f3cb160b8e169b581f6b98689dc713acad"
+
+  url "https://github.com/micronaut-projects/micronaut-starter/archive/v2.0.0.tar.gz"
+  sha256 "5e30fc1878c9d7da9046745a5f71c611fc9bd1b765e32dd5e597a6fae1739f6c"
   license "Apache-2.0"
 
   depends_on "gradle" => :build
   depends_on "openjdk"
 
   def install
-    system "gradle", "build"
+    system "gradle", "micronaut-cli:assemble", "-x", "test"
```

Comment:
This is really formula-specific step :D i wonder if they should make a new `gradle task` to abstract that away. :)